drivers/adc/ad7779/ad7779.c

```diff
@@ -635,7 +706,16 @@ int32_t ad7779_set_power_mode(ad7779_dev *dev,
 int32_t ad7779_get_power_mode(ad7779_dev *dev,
 			      ad7779_pwr_mode *pwr_mode)
 {
-	*pwr_mode = dev->pwr_mode;
+	int32_t ret;
+	uint8_t reg_data;
+
+	if (!dev->read_from_cache) {
+		ret = ad7779_spi_int_reg_read(dev, AD7779_REG_GENERAL_USER_CONFIG_1, &reg_data);
+		if (ret)
+			return ret;
+		*pwr_mode = no_os_field_get(AD7779_MOD_POWERMODE, reg_data);
+	} else
```

Comment:
nit: if you use brackets for the `if`, then use also for `else` statement.

Reply:
Added brackets for else wherever applicable